packages/react/src/components/ImageComponent/ImageComponent.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import PropTypes from 'prop-types';
+import React from 'react';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { settings } from 'carbon-components';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ *  sorts media query min-widths order to ensure the
+ * browser returns the proper sources and the specified widths
+ *
+ * @param {Array} sources image sources and min-widths
+ *
+ * @returns {Array} sorted array of sources
+ */
+const sortSources = sources => {
+  return sources.sort((a, b) => (a.minWidth > b.minWidth ? -1 : 1));
+};
+
+/**
+ * renders background image
+ *
+ * @param {object} props props object
+ * @param {object} props.images array of images used for diff breakpoints
+ * @param {string} props.defaultImage default image (usually image for largest breakpoint)
+ * @param {string} props.alt alt of the image
+ * @returns {*} picture element
+ */
+const Image = ({ images, defaultImage, alt }) => {
+  const sortedImages = sortSources(images);
```

Comment:
Is the order of these an important factor? I thought if the media doesn't match the browser will just go onto the next one, but maybe I missed something in my understanding.

Reply:
No, my understanding is that order isn't important.

Reply:
@jeffchew @photodow I am not sure about the order of these but i will update them in order ....

Reply:
the order does matter for the <picture> element!

Reply:
@annawen1 I think I see now. I had to code it out to visualize it. haha

https://codepen.io/photodow/pen/RwNMJBx?editors=1000

If you use `min and max` (not sure about browser support) it doesn't matter as much as long as you're consistent, but if you use either `min`, or `max` then you have to be more conscientious about it.